cmd/registry.go

```diff
@@ -32,31 +32,64 @@ type Registry struct {
 	Specs  []*bundle.Spec
 }
 
+// Default registry configuration section
+var defaultLocalOpenShiftConfig = registries.Config{
+	Namespaces: []string{"openshift"},
+	Name:       "lo-openshift",
+	Type:       "local_openshift",
+	WhiteList:  []string{".*$"},
+}
+
+var defaultDockerHubConfig = registries.Config{
+	Name:      "dockerhub",
+	Type:      "dockerhub",
+	URL:       "docker.io",
+	Org:       "ansibleplaybookbundle",
+	WhiteList: []string{".*-apb$"},
```

Comment:
I wonder if it makes sense to have a default whitelist of "apb" for all dockerhub repos. 

Is there any downside to making this more permissive?

Reply:
You're right, no downside. Just goes faster when an org like ours has a ton of extra images. I'll remove it.